### PR TITLE
DAOS-5851 tests: add RP_XSF rebuild

### PR DESCRIPTION
**Pull request by wangdi1**
1. Add RP_XSF tests.
2. Ouput rebuild/reintegrate/drain/extend operation
by console.

Signed-off-by: Di Wang <di.wang@intel.com>

### CODE REVIEW

File: src/rebuild/srv.c [hunk 1]
```diff
@@ -1456,14 +1459,13 @@ ds_rebuild_leader_stop_all()
...
-			  const uint32_t map_ver,
+rebuild_print_list_update(const uuid_t uuid, const uint32_t map_ver,
 			  daos_rebuild_opc_t rebuild_op,
 			  struct pool_target_id_list *tgts) {
 	int i;
 
-	D_PRINT("%s (pool="DF_UUID" ver=%u, op=%s) tgts=", str, DP_UUID(uuid),
-		map_ver, RB_OP_STR(rebuild_op));
+	D_PRINT("%s queued (pool="DF_UUID" ver=%u) tgts=",
+		RB_OP_STR(rebuild_op), DP_UUID(uuid), map_ver);
```
Comment (byron-marohn) [inline src/rebuild/srv.c:1468]: Maybe `[queued]` instead of just `queued` so it's consistent with the others? Would make it easier to search for.

**Review by byron-marohn — APPROVED** [review]

File: src/rebuild/srv.c (hunk 1 above)
Comment (wangdi1) [reply src/rebuild/srv.c]: ah, ok

**Review by byron-marohn — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Comment by johannlombardi** [comment]
Is the osa online drain failure known?

**Comment by wangdi1** [comment]
> Is the osa online drain failure known?

yes, it is known failure. I do not have ticket number, but we saw some checksum failure on the client side log.

**Pull request merged**